relay-server/src/utils/actix.rs

```diff
@@ -1,23 +1,11 @@
-use actix::System;
 use tokio::runtime::Runtime;
```

Comment:
Could we, please, also rename this file, or move `create_runtime` directly to `utils`, since there is nothing from `actix` here anymore.